src/Fetch-API.js

```diff
@@ -0,0 +1,64 @@
+let fetchApi = {
+
+  fetchCurrentTravelerData(travelerID) {
+    let fetchedCurrentTravelerData = fetch('http://localhost:3001/api/v1/travelers/${travelerID}')
+      .then(response => response.json())
+      .then(data => data.travelers.find(traveler => traveler.id === travelerID))
+      .catch(error => console.log(error.message));
+    return fetchedCurrentTravelerData;
+  },
+
+  fetchAllTravelersData() {
+    let fetchedAllTravelersData = fetch('http://localhost:3001/api/v1/travelers')
+      .then(response => response.json())
+      .then(data => data.travelers)
+      .catch(error => console.log(error.message));
+    return fetchedAllTravelersData;
+  },
+
+  fetchTripsData() {
+    let fetchedTripsData = fetch('http://localhost:3001/api/v1/trips')
+      .then(response => response.json())
+      .then(data => data.trips)
+      .catch(error => console.log(error.message));
+    return fetchedTripsData;
+  },
+
+  fetchDestinationsData() {
+    let fetchedDestinationsData = fetch('http://localhost:3001/api/v1/destinations')
+      .then(response => response.json())
+      .then(data => data.destinations)
+      .catch(error => console.log(error.message));
+    return fetchedDestinationsData;
+  },
+
+  postNewTrip(tripRequest, tripDate) {
+    fetch('http://localhost:3001/api/v1/trips', {
+      method: 'POST',
+      headers: {
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify({
```

Comment:
I see userID twice here! And also wherever you're calling this function (probs index.js) is where I pulled in the allTrips length + 1 and send it as an argument to the postNewTrip function.

Reply:
Thanks for this helpful note, Matt!